stores/metadata.go

```diff
@@ -1395,7 +1448,17 @@ func (ss *SQLStore) UpdateSlab(ctx context.Context, s object.Slab, contractSet s
 }
 
 func (s *SQLStore) RefreshHealth(ctx context.Context) error {
-	healthQuery := s.db.Raw(`
+	var nSlabs int64
+	if err := s.db.Model(&dbSlab{}).Count(&nSlabs).Error; err != nil {
+		return err
+	}
+	if nSlabs == 0 {
```

Comment:
It's not that it is slow but that the SQLite query will log an error if the subquery returns no result. The `COALESCE` we had before actually caused every slab to be updated every time since it would just cause every slab that was not returned by the subquery to have full health.